test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/IntegerTest.scala

```diff
@@ -0,0 +1,542 @@
+    assertEquals(14, Integer.numberOfTrailingZeros(1689927680))
+    assertEquals(31, Integer.numberOfTrailingZeros(-2147483648))
+    assertEquals(8, Integer.numberOfTrailingZeros(-1499141888))
+    assertEquals(3, Integer.numberOfTrailingZeros(1441572168))
+    assertEquals(30, Integer.numberOfTrailingZeros(-1073741824))
+    assertEquals(10, Integer.numberOfTrailingZeros(-1175759872))
+    assertEquals(19, Integer.numberOfTrailingZeros(781713408))
+    assertEquals(30, Integer.numberOfTrailingZeros(1073741824))
+    assertEquals(7, Integer.numberOfTrailingZeros(117264512))
+    assertEquals(1, Integer.numberOfTrailingZeros(-808865046))
+    assertEquals(23, Integer.numberOfTrailingZeros(-394264576))
+    assertEquals(26, Integer.numberOfTrailingZeros(-1946157056))
+    assertEquals(23, Integer.numberOfTrailingZeros(-1619001344))
+    assertEquals(16, Integer.numberOfTrailingZeros(-1236860928))
+    assertEquals(5, Integer.numberOfTrailingZeros(-134639968))
+    assertEquals(26, Integer.numberOfTrailingZeros(-1946157056))
+    assertEquals(28, Integer.numberOfTrailingZeros(-1342177280))
+    assertEquals(29, Integer.numberOfTrailingZeros(1610612736))
+    assertEquals(28, Integer.numberOfTrailingZeros(1879048192))
+    assertEquals(30, Integer.numberOfTrailingZeros(-1073741824))
+    assertEquals(21, Integer.numberOfTrailingZeros(1994391552))
+    assertEquals(24, Integer.numberOfTrailingZeros(-1862270976))
+    assertEquals(18, Integer.numberOfTrailingZeros(-2055471104))
+    assertEquals(20, Integer.numberOfTrailingZeros(349175808))
+    assertEquals(16, Integer.numberOfTrailingZeros(-1247477760))
+    assertEquals(12, Integer.numberOfTrailingZeros(-1874415616))
+    assertEquals(16, Integer.numberOfTrailingZeros(1336606720))
+    assertEquals(0, Integer.numberOfTrailingZeros(565581745))
+    assertEquals(31, Integer.numberOfTrailingZeros(-2147483648))
+    assertEquals(23, Integer.numberOfTrailingZeros(562036736))
+    assertEquals(3, Integer.numberOfTrailingZeros(1372687704))
+    assertEquals(15, Integer.numberOfTrailingZeros(1147437056))
+    assertEquals(10, Integer.numberOfTrailingZeros(941499392))
+    assertEquals(9, Integer.numberOfTrailingZeros(938859008))
+    assertEquals(2, Integer.numberOfTrailingZeros(1987222364))
+    assertEquals(24, Integer.numberOfTrailingZeros(251658240))
+    assertEquals(3, Integer.numberOfTrailingZeros(1598124120))
+    assertEquals(14, Integer.numberOfTrailingZeros(-1177763840))
+    assertEquals(28, Integer.numberOfTrailingZeros(1879048192))
+    assertEquals(27, Integer.numberOfTrailingZeros(-939524096))
+    assertEquals(21, Integer.numberOfTrailingZeros(-622854144))
+    assertEquals(25, Integer.numberOfTrailingZeros(-1577058304))
+    assertEquals(16, Integer.numberOfTrailingZeros(1751711744))
+    assertEquals(24, Integer.numberOfTrailingZeros(-1392508928))
+    assertEquals(19, Integer.numberOfTrailingZeros(303562752))
+  }
+
+  @Test def toBinaryString_for_values_in_range(): Unit = {
```

Comment:
`_for_values_in_range` hasn't had any meaning since 0.5.0-M1 :p Remove it. (two more below)